spec/spec_tests/sdam_spec.rb

```diff
@@ -93,9 +93,21 @@
             end
 
             it 'raises an UnsupportedFeatures error' do
+              # Need to ensure the servers are polled by monitoring thread
+              # prior to performing the following operations
+              deadline = Time.now + 5
```

Comment:
In general, I'm not fond of using manual countdowns/sleeps as a workaround for races in tests, as they tend to be rather fragile. Is there a way to achive this without relying on the event occuring within a defined period of time (e.g. by monkey patching the method that does the polling)?

Reply:
I am not sure what you mean but I think I can take that out.